internal/usage/usage.go

```diff
@@ -118,6 +118,7 @@ const (
 	PartitionFields                 = "Fields to use to partition data. You can specify up to two frequently queried fields to use for partitioning data."
 	ArchiveAfter                    = "Number of days that specifies the age limit for the data in the live Atlas cluster."
 	TargetProjectID                 = "Unique identifier of the project that contains the destination cluster for the restore job."
+	DeleteAfterDate					= "Timestamp in ISO 8601 date and time format in UTC after which Atlas deletes the user."
```

Comment:
there's already a `DeleteAfter` here but it makes reference to ip whitelists, so could we raname them both to 
`WhiteListsDeleteAfter` and `BDUsersDeleteAfter`